src/config.rs

```diff
@@ -75,6 +79,7 @@ impl PbftConfig {
 /// + `sawtooth.consensus.pbft.view_change_timeout` (optional, default 4000 ms)
 /// + `sawtooth.consensus.pbft.message_timeout` (optional, default 100 blocks)
 /// + `sawtooth.consensus.pbft.max_log_size` (optional, default 1000 messages)
+/// + `sawtooth.consensus.pbft.storage` (optional, default `"memory"`)
```

Comment:
I do not think storage type should be an on-chain setting.

Reply:
I borrowed this from where it looked like Raft was going, in that it has storage configuration wrapped up in the `Config` class. Are you thinking of configuring this with a CLI or config file parameter? If config file, would it make sense to have the SDK handle that?

Reply:
I don't think the storage type should be a run-time configuration. I think the modularity is helpful to allow us to swap out different implementations later on more easily.